configs/config.novacustom_nv4x

```diff
@@ -0,0 +1,34 @@
+CONFIG_LOCALVERSION="1.2.0"
+CONFIG_USE_OPTION_TABLE=y
+CONFIG_TSEG_STAGE_CACHE=y
+CONFIG_VENDOR_NOVACUSTOM=y
+CONFIG_MAINBOARD_SMBIOS_MANUFACTURER="Notebook"
+CONFIG_BOARD_NOVACUSTOM_NV4X=y
+CONFIG_PCIEXP_HOTPLUG_BUSES=42
+CONFIG_PCIEXP_HOTPLUG_MEM=0xc200000
+CONFIG_PCIEXP_HOTPLUG_PREFETCH_MEM=0x1c00000
+CONFIG_USE_PM_ACPI_TIMER=y
+CONFIG_CONSOLE_CBMEM_BUFFER_SIZE=0x30000
+CONFIG_MAINBOARD_SMBIOS_PRODUCT_NAME="NV4XMB,ME,MZ"
+CONFIG_UART_PCI_ADDR=0x0
+CONFIG_PCIEXP_HOTPLUG_PREFETCH_MEM_BELOW_4G=y
+CONFIG_PCIEXP_HOTPLUG_IO=0x2000
+CONFIG_SUBSYSTEM_VENDOR_ID=0x0000
+CONFIG_SUBSYSTEM_DEVICE_ID=0x0000
+CONFIG_SMMSTORE_V2=y
+CONFIG_SMMSTORE_SIZE=0x40000
+# CONFIG_TPM_PPI is not set
```

Comment:
Why PPI is not set? On MSI I noticed it helped with get-tpm command on Windows somehow.
I have added PPI support to our payload not so long ago.

Reply:
Done, although TPM seems to work fine without it on NV4x

Reply:
Yes it should.